packages/shared/src/lib/featureManagement.ts

```diff
@@ -54,6 +55,7 @@ const feature = {
   searchGoogle: new Feature('search_google', false),
   onboardingFlip: new Feature('onboarding_flip', false),
   improvedSharedPostCard: new Feature('improved_shared_post_card', false),
+  shortcutsUI: new Feature('shortcuts_ui', ShortcutsUIExperiment.Control),
```

Comment:
Added in growthbook https://app.growthbook.io/features/shortcuts_ui

Reply:
thank you!